bindings/java/src/main/com/apple/foundationdb/FDBTransaction.java

```diff
@@ -282,6 +284,17 @@ public void setReadVersion(long version) {
 		return this.getEstimatedRangeSizeBytes(range.begin, range.end);
 	}
 
+	///////////////////
+	//  Feature Options
+	///////////////////
+
+	/**
+	 * Use DirectByteBuffers to fetch getRange() results.
+	 */
+	public void setDirectBufferQueryEnabled(boolean v) {
+		enableDirectBufferQueries = v;
```

Comment:
Hm, note that as `FDBTransaction` is package-private, even though this method is `public`, I don't believe there's anyway for an outside consumer to enable this feature. It might be the case that this method would need to go on either the `Database` or `Transaction` interface or onto the `FDB` singleton or something (like `setUnclosedWarning`). The most "reasonable" place for it to go, I think, would be the `TransactionOptions`, but that doesn't work with how that class is generated.

Reply:
I've moved it to FDB. I think might be useful to set it for individual Transactions as well.